packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -74,6 +81,42 @@ export type TabsProps = {
   onChange?: MuiTabListProps["onChange"];
 };
 
+const TabLabel = ({
+  label,
+  notificationCount,
```

Comment:
I'm curious as to how `notificationCount` can be anything other than a number, as I'm seeing below on line 101? I don't know it's actual type, as it's not showing in the PR diff, but I'm assuming it can be string or something else? Is there a reason why we don't want it to exclusively a number?

Reply:
@chrispulsinelli-okta  It can't be but, ran into an issue when `notificationCount === 0`. Just checking for it's existence and that it's greater than `0` still renders the `0` for some reason... but not the badge 
<img width="437" alt="image" src="https://github.com/okta/odyssey/assets/148223016/7757fe95-daa0-4923-b130-4fe3b799723e">
Open to other suggestions. This is just the first thing I tried that worked to be honest